lib/pbench/agent/tool_meister_start.py

```diff
@@ -412,11 +411,9 @@ class ToolDataSink(BaseServer):
     def start(
         self,
         exec_dir: Path,
-        full_hostname: str,
         tds_param_key: str,
         redis_server: RedisServerCommon,
         redis_client: redis.Redis,
-        logger: logging.Logger,
```

Comment:
If we are going to squash the two commits in this PR together when we merge this PR then this is fine.  But if we are going to keep these two commits separate, then it seems we can't remove the `logger` argument unless also remove it from lines 449 and 490.  Perhaps just moving this change to the following PR would be sufficient?

Reply:
My intention was to post (just) the second commit as a PR against the `b0.71` branch and then squash-and-merge this branch on to `main`...but, I'm open to other suggestions.

Reply:
So squash and merge it is.